configure

```diff
@@ -990,4 +993,11 @@ else:
 
 gyp_args += args
 
+if warned:
+  if os.isatty(1):
+    msg = '\033[1m\033[93mCareful! Warnings happened during \'configure\'.\033[0m'
+  else:
+    msg = 'Careful! Warnings happened during \'configure\'.'
+  print msg
```

Comment:
I could, but any suggestions on what to print?

[WARNING] Warnings happend

would be a bit redundant.